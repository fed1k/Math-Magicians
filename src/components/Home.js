import React from "react";
const Home = () => {
    return (
        <div className="home-link-div">
            <h2 className="header-text">Welcome to my page</h2>
            <p className="welcome-paragraph">Lorem ipsum dolor sit amet consectetur adipisicing elit. Rerum quidem facere perspiciatis 
                natus iusto tempora nobis nihil! Saepe adipisci provident beatae rerum doloremque, illo 
                consectetur cum sit veritatis dolore eos dolor soluta sunt quas ab, facere suscipit. 
                earum rem animi placeat autem quam et, enim, ex dolores esse,natus, autem molestias eius 
                Veritatis perspiciatis vel rerum! eaque quis illum dignissimos
            </p>
        </div>
    );
}
 
export default Home;