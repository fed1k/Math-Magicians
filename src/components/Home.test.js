import React from "react";
import Home from "./Home";
import renderer from 'react-test-renderer';

it('should render correctly', ()=>{
    const tree = renderer
        .create(<Home />)
        .toJSON();
    expect(tree).toMatchSnapshot();
})