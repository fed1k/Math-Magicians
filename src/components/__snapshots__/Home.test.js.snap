// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div
  className="home-link-div"
>
  <h2
    className="header-text"
  >
    Welcome to my page
  </h2>
  <p
    className="welcome-paragraph"
  >
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Rerum quidem facere perspiciatis natus iusto tempora nobis nihil! Saepe adipisci provident beatae rerum doloremque, illo consectetur cum sit veritatis dolore eos dolor soluta sunt quas ab, facere suscipit. earum rem animi placeat autem quam et, enim, ex dolores esse,natus, autem molestias eius Veritatis perspiciatis vel rerum! eaque quis illum dignissimos
  </p>
</div>
`;
